miner/payload_building.go

```diff
@@ -132,15 +132,15 @@ func (payload *Payload) resolveBestFullPayload(payloads []*Payload) {
 			}
 
 			if payload.full == nil || payload.fullFees.Cmp(p.fullFees) < 0 {
-				log.Info("XXX updated")
+				log.Debug("best payload updated", "id", p.id)
 				payload.full = p.full
 				payload.fullFees = p.fullFees
 			}
 			p.lock.Unlock()
 		}
 	}
 
-	log.Info("XXX resolved")
+	log.Debug("best payload resolved", "id", payload.id)
```

Comment:
I wonder if we can say something more about the payload - block hash or something maybe?